charts/incubator/zerotier/values.yaml

```diff
@@ -0,0 +1,101 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: zerotier/zerotier
+  # -- image tag
+  # currently there is no zerotier manifest which
+  # allow to have a arch generic tag
+  tag: 1.8.10-arm64
```

Comment:
The tag should probably default to the amd64 variant

Reply:
why not --- change --- my default is arm64 these days

Reply:
@mabels You need to use `1.8.10` as tag for the amd64 image.